includes/Services/FlowService.php

```diff
@@ -21,8 +21,10 @@ public static function initalize_flow_data() {
 		if ( ! ( $flow_data ) ) {
 			return \update_option( Options::get_option_name( 'flow' ), $default_flow_data );
 		}
-		$updated_flow_data = self::update_flow_data_recursive( $default_flow_data, $flow_data );
-		return \update_option( Options::get_option_name( 'flow' ), $updated_flow_data );
+		if ( ! array_key_exists( 'version', $flow_data ) || strcmp( $flow_data['version'], $default_flow_data['version'] ) !== 0 ) {
```

Comment:
Since we know the exact level of nesting at which the `version` we are looking for will exist, let's use `isset` instead of `array_key_exists` which is a bit faster?